lib/fastly_nsq/listener/config.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module FastlyNsq
+  class Listener::Config
+    attr_reader :listeners
```

Comment:
I don't like the name `listeners` here because these aren't really `FastlyNsq::Listener` objects. This is an array of topic->processor class mappings. `processors`? `topics`? `topic_mapping`?